packages/mds-ingest-service/repository/index.ts

```diff
@@ -101,6 +104,16 @@ class IngestReadWriteRepository extends ReadWriteRepository {
     }
   }
 
+  public getDevices = async (ids: UUID[]): Promise<DeviceDomainModel[]> => {
+    try {
+      const connection = await this.connect('ro')
+      const entities = await connection.getRepository(DeviceEntity).find({ where: { device_id: In(ids) } })
```

Comment:
@nessur Would this benefit from your `= ANY([])` treatment?

Reply:
thanks! TypeOrm has a wrapper method for that, even :)